index.html

```diff
@@ -291,6 +292,109 @@ <h2>About me</h2>
       </div>
     </main>
 
+    <footer class="footer">
+      <section id="contact" class="container contact">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+        <form
+          action="https://formspree.io/f/mvodnloy"
+          method="post"
+          class="form"
+        >
+          <input
+            name="fullname"
+            type="text"
+            placeholder="John Doe"
+            maxlength="30"
+            aria-required="true"
+            required
+          />
+          <input
+            name="email"
+            type="email"
+            placeholder="email@example.com"
+            aria-required="true"
+            required
+            pattern="^[a-z0-9]+@(.+)$"
+          />
+          <textarea
+            name="message"
+            placeholder="write your message here"
+            maxlength="500"
+            aria-required="true"
+            required
+          ></textarea>
```

Comment:
Kindly correct the placeholder on the Textarea input test to " Hello! Glad to see you" as the Figma design looks